shared/message/vision/Ball.proto

```diff
@@ -26,28 +26,80 @@ import "Vector.proto";
 import "message/Geometry.proto";
 import "Matrix.proto";
 
+/**
+ * Contains ball measurement details.
+ */
 message Ball {
+
+    /**
+     * The technique used to make the measurement.
+     */
     enum MeasurementType {
         UNKNOWN      = 0;
         REPROJECTION = 1;
         WIDTH_BASED  = 2;
     }
+
+    /**
+     * An instance of a ball measurement with covariance.
+     */
     message Measurement {
+
+        /**
+         * The technique used to make the measurement.
+         */
```

Comment:
Use triple-slash style and try to not expand the whitespace so much, this applies to all the others
```suggestion
        /// The technique used to make the measurement.
```